app/views/layouts/footer.html.erb

```diff
@@ -0,0 +1,17 @@
+<footer>
+  <div class="row">
+    <div class="branding-section col-4 d-none d-md-flex align-items-center justify-content-center">
+      GIVING
+      <br>
+      PROFILES
+    </div>
+    <div class="copyright-section col-md-4 col-5 d-flex align-items-end justify-content-center row">
+      <div class="col-12 text-center">
```

Comment:
Seems weird to have another row and col-12 in there. Do we need that?

Reply:
took it out